rust/headless-client/src/dns_control/windows.rs

```diff
@@ -125,9 +128,9 @@ fn activate(dns_config: &[IpAddr]) -> Result<()> {
     key.set_value("Comment", &FZ_MAGIC)?;
     key.set_value("ConfigOptions", &0x8u32)?;
     key.set_value("DisplayName", &"Firezone SplitDNS")?;
-    key.set_value("GenericDNSServers", &dns_config_string)?;
+    key.set_value("GenericDNSServers", &dns_servers)?;
     key.set_value("IPSECCARestriction", &"")?;
-    key.set_value("Name", &vec!["."])?;
+    key.set_value("Name", &search_domains)?;
```

Comment:
Here's where the dot was introduced as part of a PR that added NRPT rules to fix a "DNS leak": https://github.com/firezone/firezone/pull/4530/files#diff-c84657356c1a95710239a78a52aaf64ec07bd567c7853f4e66b3bbbf97f47802R77-R78

Again, there isn't an explanation of how it works, so if we re-test it and everything works fine, I'll believe new tests over old tests. But I must have thought it was needed at the time.